src/libos/src/entry.rs

```diff
@@ -23,6 +25,7 @@ lazy_static! {
     static ref HAS_INIT: AtomicBool = AtomicBool::new(false);
     pub static ref ENTRY_POINTS: RwLock<Vec<PathBuf>> =
         RwLock::new(config::LIBOS_CONFIG.entry_points.clone());
+    pub static ref RESOLV_CONF_STR: RwLock<String> = RwLock::new(String::new());
```

Comment:
Suggestion:
```
pub static ref RESOLV_CONF_STR: RwLock<Option<String>> = RwLock::new(None);
```
Then you can initialize the `RESOLV_CONF_STR` with a valid string. After using the `RESOLV_CONF_STR`, you can just set it to `None` to make the buffer of string released.

Reply:
This make senses. The size of a resolve.conf file could be large. Releasing the `String` after it is no longer useful is good.